client/sharkey_client.go

```diff
@@ -237,3 +216,21 @@ func buildConfig(caBundlePath string) (*tls.Config, error) {
 		MinVersion: tls.VersionTLS12,
 	}, nil
 }
+
+func (c *context) shellOut(command []string) {
+	if c.conf.Sudo {
+		command = append([]string{"/usr/bin/sudo"}, command...)
```

Comment:
looking at the Go code, I think we'd have to look it up using PATH

We could pass the path in c.conf.Sudo instead of it just being a bool